docs/advanced/bootloader.md

```diff
@@ -0,0 +1,89 @@
+# Bootloader
+
+This guide explains how to create a bootloader, how to create a main program to go with the bootloader and how you can use this bootloader to perform software updates.
+
+## Creating the bootloader
+
+Creating a bootloader is similar to creating a regular application. The only additional step you need is to specify the size of the bootloader in mbed_app.json.
+
+Specify the size of the bootloader as a target override in mbed_app.json in the target field "target.restrict_size":
+
+```
+    "target_overrides": {
+        ...
+        "NUCLEO_F429ZI": {
+            "target.restrict_size": "0x20000"
+        },
+        ...
+```
+
+Adding this field:
```

Comment:
You'll need a blank line between "Adding this field:" and the top bullet or the list to be correctly rendered on docs.mbed